package/base-files/files/etc/init.d/led

```diff
@@ -53,70 +53,70 @@ load_led() {
 			"$sysfs" \
 			"$(sed -ne 's/^.*\[\(.*\)\].*$/\1/p' /sys/class/leds/${sysfs}/trigger)" \
 			"$(cat /sys/class/leds/${sysfs}/brightness)" \
-				>> /var/run/led.state
+			>> /var/run/led.state
 
-		[ "$default" = 0 ] &&
-			echo 0 >/sys/class/leds/${sysfs}/brightness
+		[ "$default" = 0 ] \
+			&& echo 0 > /sys/class/leds/${sysfs}/brightness
```

Comment:
See above. I'm not a fan of this wrapping logic, and, more importantly, it's against the style we use everywhere else.

There are similar cases below which I won't highlight anymore ...